corehq/apps/api/resources/v0_1.py

```diff
@@ -270,7 +270,7 @@ class Meta(UserResource.Meta):
 
     def obj_get_list(self, bundle, **kwargs):
         domain = kwargs['domain']
-        username = bundle.request.GET.get('username')
+        username = bundle.request.GET.get('web_username')
```

Comment:
what happens if you currently use basic or digest auth with a username param? if that used to work then i think we should at least check the proxy logs and see if anyone's doing that. should be quite easy to see and if it is in use then notify people who are using it